.travis.yml

```diff
@@ -8,6 +8,7 @@ addons:
     packages:
       - realpath
       - ruby
+      - socat
```

Comment:
Where is `socat` used? I can not see the need for it from the commit message.

Reply:
This is needed for the helm chart. as this was the one for the requirement for helm chart and kubernetes didn't mention that in the commit message

Reply:
@nixpanic do you want it as a separate commit?

Reply:
updated the commit message for socat